test/integration/grpc_management_plane_api_test.go

```diff
@@ -228,11 +240,18 @@ func TestGrpc_ConfigApply(t *testing.T) {
 
 		performConfigApply(t, nginxInstanceID)
 
-		responses = getManagementPlaneResponses(t, 3)
-		t.Logf("Config apply responses: %v", responses)
-
-		assert.Equal(t, mpi.CommandResponse_COMMAND_STATUS_OK, responses[2].GetCommandResponse().GetStatus())
-		assert.Equal(t, "Config apply successful", responses[2].GetCommandResponse().GetMessage())
+		if instanceType == "OSS" {
+			responses = getManagementPlaneResponses(t, 3)
+			t.Logf("Config apply responses: %v", responses)
+
+			assert.Equal(t, mpi.CommandResponse_COMMAND_STATUS_OK, responses[2].GetCommandResponse().GetStatus())
+			assert.Equal(t, "Config apply successful", responses[2].GetCommandResponse().GetMessage())
+		} else {
+			responses = getManagementPlaneResponses(t, 5)
```

Comment:
Cant you validate the other 2 messages that you get back and maybe add a comment explaining why there is extra messages for plus